### DISCUSSION TITLE
Installing packages status

### DISCUSSION THREAD

**Discussion by Haleshot** [marimo-team/marimo#1623] [Ideas]
Have been experimenting with a lot of marimo notebooks as of late, especially with the existing ones in marimo repo examples. Running the notebook under the examples folder with the `marimo edit` command launched the notebooks; liked the install dependencies pop up indicating to install the required packages for that notebook. However, I found myself navigating to my terminal frequently (especially for bigger packages like torch) for detailed status regarding the installation (how long it will take, etc.).

I know that the installation package popup updates till the libraries install successfully and for small packages this happens easily; for bigger packages however, it would be nice to have some sort of UI element indicating the time for installation of the packages next to the install status icon.

**Comment by mscolnick** [comment]
Are you looking for the time-elapsed or the total time? I am not sure how we would know the total time.

**Reply by Haleshot** [reply]
> Are you looking for the time-elapsed or the total time? I am not sure how we would know the total time.

I believe it would be time elapsed? Basically, the way the terminal shows detailed insights on the installation of the packages; the progress bar along with time remaining for completion, etc.
